cmd/snapshot.go

```diff
@@ -93,3 +100,50 @@ func promoteCmd() *cobra.Command {
 	cmd.MarkFlagRequired("id")
 	return cmd
 }
+
+// printCmd creates a new cobra command for upload
+func printCmd() *cobra.Command {
+	var host string
+	var req stencilv1.DownloadDescriptorRequest
+	cmd := &cobra.Command{
+		Use:  "print",
+		Args: cobra.NoArgs,
+		Annotations: map[string]string{
+			"group:core": "true",
+		},
+		RunE: func(cmd *cobra.Command, args []string) error {
+			conn, err := grpc.Dial(host, grpc.WithInsecure())
+			if err != nil {
+				return err
+			}
+			defer conn.Close()
+			client := stencilv1.NewStencilServiceClient(conn)
+			res, err := client.DownloadDescriptor(context.Background(), &req)
+			if err != nil {
+				return err
+			}
+
+			fds := &descriptorpb.FileDescriptorSet{}
+			if err := proto.Unmarshal(res.GetData(), fds); err != nil {
+				return fmt.Errorf("descriptor set file is not valid. %w", err)
+			}
+
+			fd, err := desc.CreateFileDescriptorFromSet(fds)
```

Comment:
`fd` contains only one file descriptor. This is not printing all file details.

Reply:
Should be working now, it also supports printing them to the filesystem via an `output` flag.